css/scss/pages/_publikationer.scss

```
@@ -0,0 +1,115 @@
+#local-header {
+  .full-width-section {
+	grid-area: full-width;
+  }
+
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 1rem;
+  grid-template-areas:
+	"."
+	"."
+	"full-width";
+
+  section {
+	background: $smoke;
+	padding: 1.375rem 1.25rem;
+  }
+}
+
+#article-section {
+  .full-width-section {
+	grid-area: full-width;
+  }
+
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 1rem;
+  grid-template-areas:
+	"."
+	"."
+	"full-width";
+
+  section {
+	background: $smoke;
+	padding: 1.375rem 1.25rem;
+	display: grid;
+	grid-template-columns: 1fr;
+	row-gap: 1.5rem;
+	grid-template-areas: "."
+  }
+
+  article {
+	  background: $dark-smoke;
+	  padding: 1rem;
+  }
+}
+
+#popular-tags {
+  padding: 0;
+
+  .main-tags {
+	position: relative;
+	background: $black;
+	overflow: hidden;
+
+	div {
+	  position: relative;
+	  z-index: 2;
+	  color: $white;
+
+	  p:last-child {
+		margin-bottom: 0;
+	  }
+	}
+
+	img {
+	  position: absolute;
+	  top: 0;
+	  left: 0;
+	  width: 100%;
+	  height: 100%;
+	  object-fit: cover;
+	  opacity: 0.5;
+	  z-index: 1;
+	}
+  }
+  /*
+  .more-news {
+	display: grid;
+	row-gap: 2rem;
+
+	.lvl2-news {
+	  background: $smoke;
+	  padding: 1.375rem 1.25rem;
+	}
+  }
+  */
+}
+
+@include media-lg {
+  #main-section {
+	grid-template-columns: repeat(2, 1fr);
+	column-gap: 2rem;
+	row-gap: 2rem;
+	grid-template-areas:
+	  ". ."
+	  "full-width full-width";
+  }
+
+  	#article-section {
+		section {
+			.pub-head {
+				grid-area: "header";
```

Should be:
```suggestion
				grid-area: header;
```
without the quotes 😉